lib/galaxy/managers/users.py

```diff
@@ -48,6 +49,17 @@
 """
 
 
+class UserModel(BaseModel):
+    """User in a transaction context."""
+    id: int = Field(title='ID', description='User ID')
+    username: str = Field(title='username', description='User username')
```

Comment:
The first letter of the title should be capitalized to display it in the docs? Otherwise, I think the title will be redundant since it will get the name of the field by default.
Would be nice to keep the consistency in the OpenAPI docs, this is something that can go to the guidelines docs as well. Maybe is a good idea to do a documentation review and formatting once all routes are documented.

Reply:
Will change. And yes - definitely +1 on documentation review, formatting all meta, docstrings, etc. for consistency.